src/test/java/com/rocketden/main/entity/RoomEntityTests.java

```diff
@@ -41,19 +50,20 @@ public void removeUserFromRoomSet() {
         Room room = new Room();
 
         User user = new User();
-        user.setNickname("test");
-        user.setId(1);
+        user.setNickname(NICKNAME);
+        user.setId(ID);
+        user.setUserId(USER_ID);
         room.addUser(user);
         
         User userToRemove = new User();
-        userToRemove.setNickname("nonexistent");
-        userToRemove.setId(2);
+        userToRemove.setNickname(NICKNAME_2);
+        userToRemove.setId(ID_2);
+        userToRemove.setUserId(USER_ID_2);
 
         assertFalse(room.removeUser(userToRemove));
         assertTrue(room.getUsers().contains(user));
 
-        userToRemove.setNickname("test");
-        assertTrue(room.removeUser(userToRemove));
+        assertTrue(room.removeUser(user));
```

Comment:
Suggestion: instead of calling `removeUser` with `user`, modify `userToRemove` to have matching `userId` and `nickname` as `user` (but still with `ID_2`), and verify that calling `removeUser` with `userToRemove` succeeds despite the different `id` values

Reply:
Done!